cpp/modmesh/universe/World.hpp

```diff
@@ -77,45 +83,44 @@ class World
     World & operator=(World &&) = delete;
     ~World() = default;
 
-    void add_vertex(vertex_type const & vertex);
-    void add_vertex(value_type x, value_type y, value_type z)
+    void add_point(point_type const & vertex)
     {
-        add_vertex(vertex_type(x, y, z));
+        m_points->append(vertex);
     }
-    size_t nvertex() const { return m_vertices.size(); }
-    vertex_type const & vertex(size_t i) const { return m_vertices[i]; }
-    vertex_type & vertex(size_t i) { return m_vertices[i]; }
-    vertex_type const & vertex_at(size_t i) const
+    void add_point(value_type x, value_type y, value_type z)
     {
-        check_size(i, m_vertices.size(), "vertex");
-        return m_vertices[i];
+        add_point(point_type(x, y, z));
     }
-    vertex_type & vertex_at(size_t i)
+    size_t npoint() const { return m_points->size(); }
+    point_type point(size_t i) const { return m_points->get(i); }
+    point_type point_at(size_t i) const
     {
-        check_size(i, m_vertices.size(), "vertex");
-        return m_vertices[i];
+        check_size(i, m_points->size(), "point");
+        return m_points->get(i);
     }
+    std::shared_ptr<point_pad_type> points() { return m_points; }
```

Comment:
Allow access to the point container.  We can consider to make it the sole interface for points.